db/migrate/20210119154820_add_request_method_and_response_headers_to_vendor_api_requests.rb

```diff
@@ -0,0 +1,6 @@
+class AddRequestMethodAndResponseHeadersToVendorAPIRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :vendor_api_requests, :request_method, :string
```

Comment:
May be worth back-filling this information, should be possible via the request path.

Reply:
@mnacos I've attempted this here https://github.com/DFE-Digital/apply-for-teacher-training/pull/3874/commits/97f5e75f237dbd2b5f449fe7de6cad4a861ef8cc#diff-5f48943dcb8a0ea9c0bbc153df01f259d328b5747eb11eae7b75627abb724c64R7